gobblin-compaction/src/main/java/gobblin/compaction/mapreduce/MRCompactorJobRunner.java

```diff
@@ -504,7 +503,7 @@ public int compareTo(MRCompactorJobRunner o) {
     return Double.compare(o.dataset.priority(), this.dataset.priority());
   }
 
-  private List<Path> getCumulativeLateFilePaths(Path lateDataDir) throws FileNotFoundException, IOException {
+  private List<Path> getApplicableFilePaths(Path lateDataDir) throws FileNotFoundException, IOException {
```

Comment:
Not sure what this method name means. Applicable to what? Please use a more meaningful name and/or add javadoc.
Also if it is not for late data, the parameter should not be called `lateDataDir`